### PR TITLE
Enable GitHub issue templates

### PR DESCRIPTION
**Pull request by jacky82226**
Enable issue templates by Github UI in ONNX according to this https://github.com/onnx/onnx/pull/2835.
Plan to add 
- bug
- question
- enhancement (feature request)
- good-first-issue
- documentation

After finishing this, I will propose another PR for [onnx/tutorials](https://github.com/onnx/tutorials).
Any feedback would be greatly appreciated.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# Feature Request
+
+### System information
+ONNX version (you are using):
+
+### What is the problem that this feature solves?
+Please detail the discrepancy with our current functionality.
+
+### Describe the feature
+Why is this feature necessary? What does it accomplish?
```
Comment (askhade) [inline .github/ISSUE_TEMPLATE/feature_request.md:20]: Also add a section for - Describe the alternatives you have considered. (A clear and concise description of any alternative solutions or features you've considered.)

**Comment by askhade** [comment]
Add a separate template for "New Operator"?

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+### Reproduction instructions
+- Describe the code to reproduce the behavior.
+```
+import onnx
+model = onnx.load('model.onnx')
+...
+```
+- Attach the ONNX model to the issue (where applicable)
+
```
Comment (askhade) [inline .github/ISSUE_TEMPLATE/bug.md:32]: Add a section for Expected Behavior

**Comment by askhade** [comment]
Are you also planning to add separate templates for god-first-issue and documentation? I don't think we needs separate templates for these 2.

**Comment by jacky82226** [comment]
> Are you also planning to add separate templates for god-first-issue and documentation? I don't think we needs separate templates for these 2.

I was planning to, but after carefully reviewing the existing templates we have, these 2 should be already covered.

**Comment by jacky82226** [comment]
> Add a separate template for "New Operator"?

Is it for adding a new operator? If so, hasn't it been covered by Feature Request?

**Comment by askhade** [comment]
> > Add a separate template for "New Operator"?
> 
> Is it for adding a new operator? If so, hasn't it been covered by Feature Request?

Yes this is for adding new operator. I was thinking of adding a separate template for this as it requires more details and is a pretty common use case. In case of new operator you want to ask whether current op can be constructed using existing onnx ops and added as a function instead etc.

**Comment by jacky82226** [comment]
I did check some issues about new operators, organize some sections which might help users and add a new template. @askhade please review it. THANKS!

**Pull request merged**